rllib/agents/trainer.py

```diff
@@ -1847,9 +1847,20 @@ def resolve_tf_settings():
         check_if_correct_nn_framework_installed()
         resolve_tf_settings()
 
-    @staticmethod
-    def _validate_config(config: PartialTrainerConfigDict,
-                         trainer_obj_or_none: Optional["Trainer"] = None):
+    @ExperimentalAPI
+    def validate_config(self, config: PartialTrainerConfigDict) -> None:
+        """Validates a given config dict for this Trainer.
+
+        Users should override this method to implement custom validation
+        behavior. It is recommended to call `super().validate_config()` in
+        this override.
```

Comment:
wait, we are not calling super().validate_config() in this function?

Reply:
This is the base method. Super (tune.Trainable) doesn't have a `validate_config`.
When users sub-class `Trainer`, they should call the super's (Trainer's) validate config to get all the logic that this default implementation already provides.

Reply:
yeah stupid me ...